modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -30,6 +30,11 @@ on {rh-openstack-first}
 endif::[]
 to use an external load balancer in place of the default load balancer.
 
+[NOTE]
+====
+MetalLB, that runs on a cluster, functions as an external load balancer.
+====
+
 [IMPORTANT]
```

Comment:
Per https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Rather than having two admonitions together here, it would be better I think to move this [NOTE] information into the general text. For example:

"You can configure an OpenShift Container Platform cluster to use an external load balancer in place of the default load balancer. MetalLB, which runs on a cluster, functions as an external load balancer."

Reply:
Yes, I did have the same thought initially. But the users were not quite sure about it, so thought may be a note would help highlighting it.